hibernate-core/src/main/java/org/hibernate/dialect/DB2Dialect.java

```diff
@@ -77,6 +75,7 @@
  */
 public class DB2Dialect extends Dialect {
 
+	private final static DatabaseVersion MINIMUM_VERSION = DatabaseVersion.make( 11, 1 );
```

Comment:
I guess you chose the version 11.1 because it simplified some code, but the issue says to remove support for versions older than 10.5. So I guess the whole class and maybe even the PR as a whole might need some rework due to this mistake.

```suggestion
	private final static DatabaseVersion MINIMUM_VERSION = DatabaseVersion.make( 10, 5 );
```

Reply:
> I guess you chose the version 11.1 because it simplified some code, 

Nah, it's probably as mundane as a plain old stupid fuckup. 

but the issue says to remove support for versions older than 10.5. So I guess the whole class and maybe even the PR as a whole might need some rework due to this mistake.
I will have to go through this with a bit of care yes